src/doc/unstable-book/src/language-features/optin-builtin-traits.md

```diff
@@ -19,12 +19,22 @@ has explictly opted out via a negative impl.
 impl !Type for Trait
 ```
 
+[#46108] added the `DynSized` trait, which is an implicit bound for all traits. Auto traits may not
```

Comment:
the new move to `auto trait` syntax would make this easier to do automatically, right?

Reply:
@nikomatsakis by "do this automatically" do you mean automatically relaxing the default `DynSized` bound for auto traits? I guess so, because with the old way (`impl  Trait for .. {}`) there is no way to tell from the trait definition whether it is an auto trait or not, whereas with `auto trait` it is obvious.